omod/src/main/webapp/resources/scripts/visit/visit.js

```diff
@@ -297,6 +297,7 @@ angular.module("visit", [ "filters", "constants", "encounterTypeConfig", "visitS
                             visit: $scope.visit,
                             encounter: $scope.encounter,
                             patient: $scope.visit.patient,
+                            breadcrumbOverride: encodeURIComponent(JSON.stringify(breadcrumbOverride)),
```

Comment:
Not directly related to this commit, but fixes an issue where breadcrumbs weren't working right in the vust note "section" context